hw/top_earlgrey/lint/top_earlgrey.waiver

```diff
@@ -12,6 +12,10 @@ set_clock_drivers prim_clock_buf prim_clock_mux2
 waive -rules RESET_MUX -location {top_earlgrey.sv} -regexp {Asynchronous reset .*rstmgr_resets\.rst.* is driven by a multiplexer} \
       -comment "This is dedicated reset infrastructure, and hence permissible"
 
+waive -rules CLOCK_MUX -location {clkmgr.sv | top_earlgrey.sv} -regexp {.*clk_io_div.* is driven by a multiplexer here} \
```

Comment:
```suggestion
waive -rules CLOCK_MUX -location {clkmgr.sv top_earlgrey.sv} -regexp {.*clk_io_div.* is driven by a multiplexer here} \
```

Guessing about syntax, but if it is a TCL list, a space should do.

Reply:
o let me give that a try.